packages/manager/apps/key-management-service/src/components/credential/create/identities/IdentitiesRootAccount.component.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { Dispatch, SetStateAction } from 'react';
+import {
+  OsdsToggle,
+  OsdsText,
+  OsdsFormField,
+  OsdsCheckbox,
+} from '@ovhcloud/ods-components/react';
+import { useTranslation } from 'react-i18next';
+import {
+  ODS_THEME_COLOR_INTENT,
+  ODS_THEME_TYPOGRAPHY_SIZE,
+} from '@ovhcloud/ods-common-theming';
+
+type IdentitiesRootAccountProps = {
+  isRootAccount: boolean;
+  setIsRootAccount: Dispatch<SetStateAction<boolean>>;
+};
+
+const IdentitiesRootAccount = ({
+  isRootAccount,
+  setIsRootAccount,
+}: IdentitiesRootAccountProps) => {
+  const { t } = useTranslation('key-management-service/credential');
+
+  return (
+    <OsdsFormField>
```

Comment:
```suggestion
    <OsdsFormField inline>
```
to prevent some extra unwanted clickable space